templates/README.md.tpl

```diff
@@ -0,0 +1,50 @@
+### Repos
```

Comment:
I feel like before getting into the repos, we should plug our website, twitter, and youtube to let them know that we have an active online presence and encourage them to join the community. Ideally once we start hosting regular events (tbd on dates) we can feature that here too so they can join in!

Reply:
these info are already included in the overview header 
![image](https://user-images.githubusercontent.com/3187948/183766527-a78d1447-1ecd-4035-b008-964d780375a7.png)